tests/common/mock_cgroup_commands.py

```diff
@@ -70,41 +67,60 @@
 
 ]
 
-__DEFAULT_FILES = [
+__service_file = "{0}.service".format(get_osutil().get_service_name())
+
+__MOCKED_FILES = [
     (r"^/proc/self/cgroup$", os.path.join(data_dir, 'cgroups', 'proc_self_cgroup')),
     (r"^/proc/[0-9]+/cgroup$", os.path.join(data_dir, 'cgroups', 'proc_pid_cgroup')),
     (r"^/sys/fs/cgroup/unified/cgroup.controllers$", os.path.join(data_dir, 'cgroups', 'sys_fs_cgroup_unified_cgroup.controllers')),
+    (r"^/lib/systemd/system/{0}$".format(__service_file), os.path.join(data_dir, 'init', __service_file + "_system-slice")),
 ]
 
-# These paths are mapped to the given tmp_dir, e.g. "/lib/systemd/system" becomes "<tmp_dir>/lib/systemd/system".
-# The mapping is done only during calls to fileutil.read_file, fileutil.write_file, fileutil.mkdir and os.path.exists.
-__SYSTEM_PATHS = [
-    "/lib/systemd/system",
-    "/etc/systemd/system"
+__MOCKED_PATHS = [
+    r"^(/lib/systemd/system)",
+    r"^(/etc/systemd/system)"
 ]
 
-
+# W0212: Access to a protected member __commands of a client class (protected-access) - Disabled: patcher.__commands is added
```

Comment:
Can we disable this for the `tests/` module? Looks like we do use it quite frequently in tests which seems like a very viable scenario

Reply:
Sounds good; let me do that on another PR

Reply:
Awesome thanks!